src/displayUI.js

```diff
@@ -6,52 +6,64 @@ import { instanceOfMainClass } from './mainClass';
 // Display one Category element
 export function displayOneCategory(categoryElement) {
 
+  let searchContent = document.querySelector("#searchBar");
+
   let category = document.createElement('div');
   category.className = 'category-main';
+  category.id = categoryElement.name;
 
   let categoryItem = document.createElement("div");
   categoryItem.textContent = categoryElement.name;
-  let firstWord = categoryElement.name;
-  let parameter = firstWord.split(" ")[0];
-  categoryItem.id = parameter;
+  let elementName = categoryElement.name;
 
   let categoryItemSvg = document.createElement('div');
   categoryItemSvg.className = "fas fa-star";
+  categoryItemSvg.id = `${elementName}Star`;
+
+  if (searchContent.value === '') {
+    category.classList.add('fadein-animation');
+  }
+
+  if (instanceOfMainClass.selectedCategory && instanceOfMainClass.selectedCategory === categoryElement.name) {
```

Comment:
Tend, to use seters, to follow enccapsulation.

Reply:
Here i've just checked if slectedCategoy is true and if its value is same as name of current element, i haven't change it. Why is it not following encapsulation?